src/test/regress/sql/multi_metadata_sync.sql

```diff
@@ -825,8 +829,23 @@ GRANT EXECUTE ON FUNCTION start_metadata_sync_to_node(text,int) TO non_super_met
 GRANT EXECUTE ON FUNCTION stop_metadata_sync_to_node(text,int,bool) TO non_super_metadata_user;
 GRANT ALL ON pg_dist_node TO non_super_metadata_user;
 GRANT ALL ON pg_dist_local_group TO non_super_metadata_user;
+GRANT ALL ON SCHEMA citus TO non_super_metadata_user;
```

Comment:
Not sure if I asked somewhere else, but why do we need all this GRANTs?

Reply:
Since we need to have access on those for the user non_super_metadata_user. Otherwise we can not pass the acl checks on worker nodes.

Reply:
Is GRANT `SELECT` enough, at least for `SCHEMA citus`?

Reply:
Nope it is now enough